Simulator/Simulator/Experiments/src/experiments/embedded_synfire_chains.c

```diff
@@ -176,20 +235,25 @@ static inline Array* create_neurons(uint32_t n_neurons, NeuronClass* n_class) {
 
 static inline Array* create_chains(Array* neuron_pool, embedded_synfire_chains_config* config) {
 	Array* chains_input_neurons = array_create(config->n_chains, 0, sizeof(Array));
-
+	uint32_t n_synapses = 0;
 	for (uint32_t chain_idx = 0; chain_idx < config->n_chains; ++chain_idx) {
 		printf("Building chain %d\n", chain_idx);
 
 		// to construct a chain start with n_k = (m + M) / 2
 		// and construct succesive layers until the number of neurons per
 		// layer is too big or too small
-		uint32_t n_k = (M + m) / 2;
+		uint32_t n_k = config->n_exci_min_neurons; //(config->n_exci_max_neurons + config->n_exci_min_neurons) / 2;
```

Comment:
Add a comment to specify that the in the original paper they used the m value but you can play with that

Reply:
Done